src/game/AI/ScriptDevAI/scripts/outland/coilfang_reservoir/serpent_shrine/serpent_shrine.cpp

```diff
@@ -375,10 +383,15 @@ void instance_serpentshrine_cavern::EngageBossConsole(uint32 uiType, GameObject*
         }
         console = instance->GetGameObject(m_goEntryGuidStore[goEntry]);
     }
+    #ifdef PRENERF_2_3
```

Comment:
ifdefs are generally formatted at the complete left, not inline ill fix it myself tho